### PR TITLE
NAS-122861 / 23.10 / add `homarr` to `community` train

### PR DESCRIPTION
**Pull request by stavros-k**

### CODE REVIEW

File: library/ix-dev/community/homarr/templates/_homarr.tpl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            - name: {{ $env.name }}
+              value: {{ $env.value }}
+            {{ end }}
+          {{ end }}
+          probes:
+            liveness:
+              enabled: true
+              type: http
+              port: "{{ .Values.homarrNetwork.webPort }}"
+              path: /api/configs
```
Comment (stavros-k) [inline library/ix-dev/community/homarr/templates/_homarr.tpl:34]:
```suggestion
              path: /api/configs
```

File: library/ix-dev/community/homarr/item.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+icon_url: https://homarr.dev/img/logo.svg
+categories:
+  - productivity
+screenshots: []
```
Comment (stavros-k) [inline library/ix-dev/community/homarr/item.yaml:4]:
```suggestion
screenshots:
  - https://homarr.dev/assets/images/add-app-modal-151ae6d1969d242acde705b33d160c0b.webp
  - https://homarr.dev/assets/images/widgets-select-type-28beaddc83c6aef18f715b034b4f38b6.webp
  - https://homarr.dev/assets/images/app-integration-d80d859a281f8bef01dbf1e2d7f95b45.webp
```

File: library/ix-dev/community/homarr/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+name: homarr
+description: Homarr is a sleek, modern dashboard that puts all of your apps and services at your fingertips.
+annotations:
+  title: Homarr
+type: application
+version: 1.0.0
+apiVersion: v2
+appVersion: '1.1.3.2982'
```
Comment (Qubad786) [inline library/ix-dev/community/homarr/Chart.yaml:8]: The `appVersion` seems to conflict with the tag version we are using in the docker image..is that expected?

Comment (stavros-k) [inline library/ix-dev/community/homarr/Chart.yaml:8]:
```suggestion
appVersion: '0.12.3'
```

Comment (stavros-k) [reply library/ix-dev/community/homarr/Chart.yaml]: Missed that, thanks!

**Review by sonicaj — APPROVED** [review]

**Pull request merged**